.github/workflows/consensus_tests.yml

```diff
@@ -64,10 +64,4 @@ jobs:
       run: go test ./x/projects/... 
 
     - name: lava subscription unit Tests
-      run: go test ./x/subscription/... 
-    
-    - name: lavad cli test
-      run: |
-        chmod +x ./scripts/cli_test.sh    
-        ./scripts/cli_test.sh     
-        killall lavad    
+      run: go test ./x/subscription/... 
```

Comment:
why don't we also run tests from the modules: downtime, dualstaking and fixationstore?

Reply:
we should, ill add it now